libraries/dotnet/EnergyOrigin.IntegrationEvents/doc/specific-integration-guides/masstransit-rabbitmq.md

```diff
@@ -0,0 +1,120 @@
+# Integration with RabbitMQ using MassTransit's Transactional Outbox Pattern
+
+## Overview
+
+Energy Track & Trace™ currently uses RabbitMQ, as the message broker for publishing and consuming integration events.
+To ensure reliability and consistency between database operations and message publishing,
+[MassTransit's](https://masstransit.io/) transactional outbox pattern is used,
+using the MassTransit.EntityFrameworkCore, and MassTransit.RabbitMQ NuGet packages.
+
+### Setup Guide
+
+To start publishing integration event messages,
+using [Transactional Outbox Configuration](https://masstransit.io/documentation/configuration/middleware/outbox),
```

Comment:
Do we need the below text when we have the masstransit guide?

Reply:
Godt spørgsmål. Jeg tænker vores rabbitmq options er rimeligt specifik for os, da det er de parametre som eo-base bruger, når den deployer rabbitmq.

Reply:
Removed: https://github.com/Energinet-DataHub/energy-origin/pull/1279/commits/72e19f5843d53b37751d65c929a13f1f7d24c78f